docs/FRSKY_R9.md

```diff
@@ -67,7 +67,24 @@ A description for installing a fan along with documentation for 3D printed case
 
 ### As Receiver ###
 
-The R9M module can also be flashed with firmware which allows it to be used as mLRS receiver. This yields a receiver working at high transmit power for longer range applications. For details please search the rcgroups thread and ask on rcgroups or discord if you have questions.
+The R9M module can also be flashed with firmware which allows it to be used as mLRS receiver. This yields a receiver working at high transmit power for longer range applications. 
+
+Connections are as follows:
+
+4-Pin Header:
```

Comment:
shiuldn't this be alreay in the R9M tx section?

Reply:
Not at the moment - does it make sense to add a 'Connections' section for the Tx?

Reply:
Added

Reply:
that's really just taste and nitpicking (jo, in certain matters I am nitpicky) ... for the R9M tx module section, the "Connections" stand a bit loose and come a bit unexpected. What about moving them to after the 3 bullets and before the tx versions sub chapter, and to make "Connections" also a sub chapter, i.e. "### Connections ###". Just a thought...

Reply:
Easy enough, other pages (like Seeed) do Connections first but can see how it makes sense here.

Reply:
it's just a thought stream uniltered by my brain ... may not make sense ...

Reply:
I somehow miss a bit an intro sentence. I liked the original one. Maybe "The R9M module can also be used as mLRS receiver, when flashed with the corresponding firmware. This yields a receiver with high transmit power for long range applications."